### DISCUSSION TITLE
Welcome to Cozie-Apple Discussions!

### DISCUSSION THREAD

**Discussion by mariofrei** [cozie-app/cozie-apple#78] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## How to get started with Cozie-Apple?
Just go to our [documentation website](https://cozie-apple.app/docs/). We have guides for app installation, app setup, and downloading the data generated by Cozie.
If you encounter any challenges, feel free to post your issues here and we will try to help you.

**Comment by kkrzewin** [comment]
Has anyone integrated SensorKit into Cozie?

**Reply by mariofrei** [reply]
We don’t have SensorKit integrated into the Cozie app, and I am not aware that anyone else has tried it.
(Apologies for the late response. I am currently traveling.)
